addons/core/functions/fnc_getVehicleRadios.sqf

```diff
@@ -0,0 +1,37 @@
+#include "script_component.hpp"
+
+/*
+    Name: TFAR_fnc_getVehicleRadios
+
+    Author(s):
+        Jaffa
+
+    Description:
+        Gets a list of radios in the specified vehicle
+
+    Parameters:
+        0: Object - vehicle
+
+    Returns:
+         Array of LR radio arrays in format: 
+             0 - Object - Vehicle, 1 - String - Radio Settings ID
+         or empty array if no vehicle radios found
+
+    Example:
+        _radios = _vehicle call TFAR_fnc_getVehicleRadios;
+*/
+params ["_vehicle"];
+
+if (isNull _vehicle || {!(_vehicle call TFAR_fnc_hasVehicleRadio)}) exitWith {[]};//Vehicle is null or vehicle doesn't have LR Radio
```

Comment:
The comment in this line is useless

Reply:
Tell Nkey that https://github.com/michail-nikolaev/task-force-arma-3-radio/blob/1.0/addons/core/functions/fnc_vehicleLr.sqf#L25

Reply:
I know, that the comment is in this function too.
But it's still useless, because it's not providing any additional information.
It's just the same as the code.